### ISSUE TITLE
The output seems quite un-normal but I cannt figure out why

### ISSUE THREAD

**Issue by edwardpixiu** [m-lyon/dMRI-RCNN#6]
Hi sir:

        I ve follow up ur work progress which u mentioned in the paper

        I have a DWIs series with 13 diffusion direction which 1 B0 and 12 random gradients.

        My Pre-processing pipeline like below:

        1. I re-arrange the data to make it only have 12 gradients components, and denoise it through Dipy piple(Patch2self)
        2. I put B0 series into FSL - FDT pipeline to choose the best b0s since i ve 3 B0 series
        3. Also do denoising for B0s series
        4. Simple way to stack B0s and denoised 12 components, so the data shape would be like 128 * 128 * 49 * 13, divded the data by 4000 since shell correspond is 1000
        5. Run the FAST pipeline based on the stacked data, get partial volume segmentation
        6. Generate the mask according to the pveseg.nii.gz 
        7. run_dmri_rcnn.py to get the output 

        But the output I get eventually seems quite wrong which I show below, I cannot figure out the reason, so am eager to receive your response if you can help me, sincerely thank u sir

![Image](https://github.com/user-attachments/assets/d1e85ef9-2f2a-49e2-af71-68e804a4f812)

**Comment by m-lyon** [comment]
So the 3D RCNN was trained on HCP data which has 1.25mm^3 spatial resolution, this mismatch in resolution could be the cause of the issue. Additionally, if you use the `-n` and `-c` flag instead of dividing by `4000` that may help also, as your data is not from the HCP and may therefore have a different contrast to the data the model was trained on.

To help diagnose the resolution issue you could resize your image to a resolution of 1.25mm^3 and retry with this data. Obviously because your data is low res the resizing won't be perfect, but it could produce better results.

If this doesn't work you could finetune the model on a small subset of your data, the instructions for which can be found [here](https://github.com/m-lyon/dMRI-RCNN?tab=readme-ov-file#training).

Let me know if you get stuck.

**Comment by edwardpixiu** [comment]
O thank u sir, the data i given into the model has 3mm^3 spatial resolution,

**Comment by edwardpixiu** [comment]
O sir, thank u, the current data I given into model is 3mm^3 spatial resolution, as you say may be is the cause, I will retry and connect with u if the stuck occur to me. 
Meanwhile I have another question, if my particular data which used to inference is for cannie brain not human beings, whether I need finetune even re-train the model to get the proper weights?

**Comment by m-lyon** [comment]
I would finetune for new non-human data yes. I wouldn't say you'd have to train from scratch.
